site/en/docs/extensions/reference/sidePanel/index.md

```diff
@@ -0,0 +1,202 @@
+---
+api: sidePanel
+---
+
+## Overview {: #overview }
+
+Chrome features a built-in side panel that enables users to view more information alongside the main content of a webpage. The SidePanel API allows extensions to add their own panels to the Chrome browser. Some features include:
```

Comment:
Can we add a screenshot of a side panel here?

Reply:
I like the first one! It takes up less space and also shows where to open it from.